diff --git a/dependencies/lua-5.3/src/luaconf.h b/dependencies/lua-5.3/src/luaconf.h
index 9eeeea69e2..6a519a73c7 100644
--- a/dependencies/lua-5.3/src/luaconf.h
+++ b/dependencies/lua-5.3/src/luaconf.h
@@ -61,14 +61,12 @@
 #if defined(LUA_USE_LINUX)
 #define LUA_USE_POSIX
 #define LUA_USE_DLOPEN		/* needs an extra library: -ldl */
-#define LUA_USE_READLINE	/* needs some extra libraries */
 #endif
 
 
 #if defined(LUA_USE_MACOSX)
 #define LUA_USE_POSIX
 #define LUA_USE_DLOPEN		/* MacOS does not need -ldl */
-#define LUA_USE_READLINE	/* needs an extra library: -lreadline */
 #endif
 
 
