diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 037b7d1..df1bcce 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -24,7 +24,6 @@ endif()

 if (LibZstd_FOUND)
     target_sources(KF5Archive PRIVATE kzstdfilter.cpp)
-    target_link_libraries(KF5Archive PRIVATE PkgConfig::LibZstd)
 endif()


@@ -72,8 +71,6 @@ target_include_directories(KF5Archive
 target_link_libraries(KF5Archive
     PUBLIC
         Qt${QT_MAJOR_VERSION}::Core
-    PRIVATE
-        ZLIB::ZLIB
 )

 set_target_properties(KF5Archive PROPERTIES
